script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -1,133 +1,133 @@
+Panicking, the woman darted out of the room calling for a doctor.
 Pitter patter pitter patter.
-I get that she's in a rush, but she really shouldn't run in a hospital.
+I got that she was in a rush, but she really shouldn't be running in a hospital.
 <Page13>
 "------"
 I couldn't raise my body, so I turned my head to look up at the window.
-My eyeballs throb as though they are about to splinter in two.
-My brain feels like it's been scrambled with a knife.
+My eyeballs throbbed as though they were about to split in two.
+My brain felt like it's been scrambled with a knife.
 <Page14>
 Truly... an unbelievable blue sky.
 More than the white rays of sunlight, it was the harshness of that unblemished blue that pained my eyes.
 <Page15>
 A few days later.
 For a while after I woke up, I couldn't move from the bed.
 <Page16>
-Looking up, I can see the lofty skies outside same as ever.
-The curtains flutter and sway.
-A dry breeze strokes my cheek.
-The stainless, immaculate blue makes my eyes spin.
+Looking up, I could see the lofty skies outside being the same as ever.
+The curtains fluttered and swayed.
+A dry breeze stroked my cheek.
+The stainless, immaculate blue made my eyes spin.
 <Page17>
-Where am I? What day is it?
-Why am I in the hospital?
-It's coming back to me, bit by bit.
+Where was I? What day was it?
+Why was I in a hospital?
+It was coming back to me, bit by bit.
 <Page18>
 All except one thing.
-I continue staring out the window, unable to shake off the feeling that I've forgotten something very important. A "promise" that's bringing me to tears.
+I continued staring through the window, unable to shake off the feeling that I had forgotten something very important. A "promise" that was bringing me to tears.
 <Page19>
 "It's nice to meet you, Tohno Shiki-kun. Congratulations on your recovery."
-Saying those words, an old man whom I've never seen before reaches out for a handshake.
+Saying those words, an old man whom I had never seen before reached out for a handshake.
 <Page20>
-His kind smile and gentle tone give me the impression of a fine adult.
-I'm sure he's someone who can swap between a smile and a straight face with the flick of a switch.
+His kind smile and gentle tone gave me the impression of a fine adult.
+I was sure he was someone who could swap between a smile and a straight face with the flick of a switch.
 His immaculate white clothes, not a thing out of place, suited him perfectly.
 <Page21>
 "Can you understand what I am saying, Shiki-kun?"
 "...No. Why am I in a hospital?"
 "You don't remember, do you? You were involved in a car accident while you were walking down the street."
 "Your chest was pierced by a shard of glass. It was unlikely that you would survive."
 <Page22>
-The old man in white says something very unbefitting of what a doctor would say, still with a smile on his face.
+The old man in white said something very unbefitting of what a doctor would say, still with a smile on his face.
```

Comment:
I think that you can drop the "would say"; unbefitting modifies a person rather than a behaviour. So, "The old man in white said something very unbefitting of a doctor, still with a smile on his face"